templates/react/README.md

```diff
@@ -0,0 +1,43 @@
+## Overview
+
+This is a template for dApp development guides that interact with the blockchain, abstracting local, mainnet, and a wallet connection
```

Comment:
Could use a full stop and also **that interact with the blockchain** is redundant for a **dApp**

Reply:
Will clean this up a bit